conf/bigearthnet.yaml

```
@@ -1,22 +1,24 @@
+module:
+  _target_: torchgeo.trainers.MultiLabelClassificationTask
+  loss: "bce"
+  model: "resnet18"
+  learning_rate: 1e-3
+  learning_rate_schedule_patience: 6
+  weights: null
+  in_channels: 14
+  num_classes: 19
+
+datamodule:
+  _target_: torchgeo.datamodules.BigEarthNetDataModule
+  root: "data/bigearthnet"
+  bands: "all"
+  num_classes: ${experiment.module.num_classes}
+  batch_size: 128
+  num_workers: 4
+
 trainer:
+  _target_: lightning.Trainer
```

```suggestion
  _target_: lightning.pytorch.Trainer
```
Some classes are aliased and some aren't. I like to be consistent and explicit.